model/clusters_mgmt/v1/azure_etcd_data_encryption_type.model

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright (c) 2025 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Contains the necessary attributes to support data encryption for Azure based clusters.
+struct AzureEtcdDataEncryption {
+	// The key management strategy used for the encryption key that encrypts the etcd data.
+	// Accepted values are: "customer_managed", "platform_managed".
+	// By default, "platform_managed" is used.
+	// Currently only "customer_managed" mode is supported.
```

Comment:
I think I understand what this means, but it is a confusing comment regardless. It says that _only_ 'customer_managed' is supported, but it also says that 'platform_managed' is the default - so we're saying that the default is not supported.

Which I guess is true, but still...

Since this is a temporary situation, I guess it counts as a nitpick.